i18n/locales/it-IT/hourofcode/how-to/index.md

```diff
@@ -13,13 +13,11 @@ Unisciti al movimento ed introduci un gruppo di studenti alla loro prima ora di
 
 ## 2. Scegli un'esercitazione
 
-Noi forniamo un'ampia scelta di divertenti [esercitazioni autogestite](%= resolve_url('/learn') %) ed altrettante [gestite dall'insegnante](%= resolve_url('https://code.org/educate/teacher-led') %) per partecipanti di tutte le età, create da diversi partner.
-
-[![](/images/fit-700/tutorials.png)](%=resolve_url('/learn') %)
+We provide a variety of fun, [student-guided tutorials](%= resolve_url('/learn') %) for all age groups and experience levels. Students do the activities on their own, though many activities include lesson plans for teachers (you'll see the link when you click the activity) to guide discussion or extend the activity. [![](/images/fit-700/tutorials.png)](%=resolve_url('/learn') %)
 
 ## 3. Promuovi la tua Ora del Codice
 
-Promuovi la tua Ora del Codice [con questi strumenti](%= resolve_url('/promote/resources') %) e incoraggia altri ad ospitare i loro eventi.
+Promote your Hour of Code [with these tools](%= resolve_url('/promote/resources') %) and encourage others to host their own events.
```

Comment:
Looks like another example where a source string changed early in the file and we lost translations later in the file. 🤷‍♀️ 

I re-committed the translations in crowdin.